includes/functions/functions_dates.php

```diff
@@ -262,13 +274,19 @@ function zen_checkdate($date_to_check, $format_string, &$date_array)
  */
 function zen_is_leap_year($year)
 {
-    if ($year % 100 == 0) {
-        if ($year % 400 == 0) return true;
-    } else {
-        if (($year % 4) == 0) return true;
+    if ($year % 4 != 0) {
+        return false;
+    }
+
+    if ($year % 100 != 0) {
+        return false;
+    }
+
+    if ($year % 400 != 0) {
+        return false;
     }
 
-    return false;
+    return true;
 }
```

Comment:
At this point, the number is divisible by 4, divisible by 100 and divisible by 400. It is a leap year. 

This return is correct.